controllers/jenkins/pipelinerun/pipelinerun_controller.go

```diff
@@ -161,25 +161,52 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
-	// requeue immediately
+	// requeue after 1 second
 	return ctrl.Result{RequeueAfter: 1 * time.Second}, nil
 }
 
-func (r *Reconciler) triggerJenkinsJob(devopsProjectName, pipelineName string, prSpec *devopsv1alpha4.PipelineRunSpec) (*job.PipelineBuild, error) {
-	boClient := job.BlueOceanClient{JenkinsCore: r.JenkinsCore, Organization: "jenkins"}
-	return boClient.Build(job.BuildOption{
-		Pipelines:  []string{devopsProjectName, pipelineName},
+func (r *Reconciler) triggerJenkinsJob(projectName, pipelineName string, prSpec *devopsv1alpha4.PipelineRunSpec) (*job.PipelineBuild, error) {
+	c := job.BlueOceanClient{JenkinsCore: r.JenkinsCore, Organization: "jenkins"}
+
+	branch, err := getBranch(prSpec)
+	if err != nil {
+		return nil, err
+	}
+
+	return c.Build(job.BuildOption{
+		Pipelines: []string{projectName, pipelineName},
 		Parameters: parameterConverter{parameters: prSpec.Parameters}.convert(),
+		Branch:    branch,
 	})
 }
 
+func getBranch(prSpec *devopsv1alpha4.PipelineRunSpec) (string, error) {
+	var branch = ""
+	if prSpec.IsMultiBranchPipeline() {
+		if prSpec.SCM == nil || prSpec.SCM.RefName == "" {
+			return "", fmt.Errorf("please configure SCM reference name and type for multi branch Pipeline")
```

Comment:
`please xxx` does look like an error message. Suggestion pattern:

* `xxx is missing`
* `xxx is wrong`
* `failed to xxx`